compact/definitions.xml

```diff
@@ -601,15 +601,20 @@ Service gaps in FW direction (before endcapP ECAL) and BW direction (before endc
     <comment> HcalEndcapN needs to clear the straight beampipe with 1 * crossing angle + its radius of 22.25 mm </comment>
     <constant name="HcalEndcapN_rmin"     value="max((HcalEndcapN_zmin + HcalEndcapN_length) * tan(abs(CrossingAngle)) + 22.25 * mm, 14.0 * cm)"/>
 
-    <constant name="HcalBarrel_thickness" value="93.0*cm"/> <!-- ref: as-built CAD, plus a little room for the longer half-plates -->
-    <constant name="HcalBarrel_rmin"      value="Solenoid_rmax"/>
-    <constant name="HcalBarrel_rmax"      value="HcalBarrel_rmin + HcalBarrel_thickness"/>
-    <constant name="HcalBarrelForward_zmax"  value="ForwardServiceGap_zmin"/>
-    <constant name="HcalBarrelBackward_zmax" value="BackwardServiceGap_zmin"/>
-    <constant name="HcalBarrel_length"    value="HcalBarrelForward_zmax + HcalBarrelBackward_zmax"/>
-    <constant name="HcalBarrel_offset"    value="(HcalBarrelForward_zmax - HcalBarrelBackward_zmax)/2"/>
+    <constant name="HcalBarrel_thickness"     value="86.18*cm"/>
```

Comment:
Can we add a reference to where these numbers are coming from? A reference to an internal document number is fine too, not suggesting we need to put this on a public document database.